transformer_engine/pytorch/export.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# See LICENSE for license information.
+
+"""Export utilities for TransformerEngine"""
+from contextlib import contextmanager
+from .constants import dist_group_type
+
+_IN_ONNX_EXPORT_MODE = False
+
+@contextmanager
+def onnx_export(
+    in_phase: bool = False,
+) -> None:
+    """
+    Context manager for exporting to ONNX.
+
+    .. code-block:: python
+
+        with onnx_export(enabled=True):
+            torch.onnx.export(model)
+
+    ----------
+    enabled: bool, default = `False`
```

Comment:
The example and this doc uses `enabled` but the real argument name is currently defined as `in_phase`.
I think `enabled` is clearer.